drivers/note/note_driver.c

```diff
@@ -2034,10 +2034,10 @@ void sched_note_filter_irq(FAR struct note_filter_named_irq_s *oldf,
 void sched_note_filter_tag(FAR struct note_filter_named_tag_s *oldf,
                            FAR struct note_filter_named_tag_s *newf)
 {
-  irqstate_t falgs;
   FAR struct note_driver_s **driver;
+  irqstate_t irq_mask;
```

Comment:
```suggestion
  irqstate_t flags;
```
like other places

Reply:
the naming style in note_driver.c is irq_mask